test/integration/roles/test_win_package/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+msi_url: https://ansible-ci-files.s3.amazonaws.com/test/integration/roles/test_win_msi/7z922-x64.msi
```

Comment:
That s3 bucket is in an Ansible AWS account. It was added as a more reliable place to put binaries for tests without having to put them in the git repo.